DigitalLearningSolutions.Web/Styles/index.scss

```diff
@@ -194,3 +194,72 @@ ul.no-bullets {
 .vertical-align-centre {
     vertical-align: middle
 }
+
+.side-nav-menu {
+  @include govuk-media-query($until: desktop) {
+    display: none;
+  }
+}
+
+.side-nav-menu-bottom {
+  @include nhsuk-responsive-padding(0, "left");
+
+  @include govuk-media-query($from: desktop) {
+    display: none;
+  }
+}
+
+.side-nav__heading {
+  font-weight: 600;
+  color: $color_nhsuk-grey-1;
+  @include nhsuk-responsive-margin(3, "bottom");
+  @include nhsuk-responsive-padding(1, "top");
+  font-size: 1.1875rem;
+  line-height: 1.25;
+
+  @include govuk-media-query($until: desktop) {
+    color: $nhsuk-text-color;
+
+    @include govuk-media-query($from: tablet) {
+      font-size: 1.5rem;
+    }
+  }
+}
+
+  .side-nav__link {
+    text-decoration: none;
+
+    &:visited {
+      color: $nhsuk-link-color;
+    }
+
+    &:hover {
+      color: $nhsuk-link-hover-color;
+      text-decoration: underline;
+    }
+  }
+
+  .side-nav__list {
+    font-weight: 400;
+    @include nhsuk-responsive-margin(0, "bottom");
+    line-height: 1.3;
+  }
+
+  .side-nav__item {
+    @include nhsuk-responsive-padding(1, "bottom");
+    @include nhsuk-responsive-padding(1, "top");
+    font-size: 1rem;
+
+    @include govuk-media-query($from: tablet) {
+      @include govuk-media-query($until: desktop) {
+        font-size: 1.1875rem;
+      }
+    }
+  }
+
+  .side-nav__item--selected {
+    border-left: 4px solid $nhsuk-link-color;
+    margin-left: -0px;

```

Comment:
0 can be unitless and I don't think we need the "-" sign

Reply:
Whoops, not sure how that happened, this should be -12px.